frontend-react/src/resources/SubmissionsResource.ts

```diff
@@ -39,7 +41,7 @@ export default class SubmissionsResource extends AuthResource {
 
     static listUrl(searchParams: SubmissionsResourceParams): string {
         return `
-        ${RS_API_URL}/api/waters/org/${searchParams.organization}/submissions?pageSize=${searchParams.pageSize}&cursor=${searchParams.cursor}&endcursor=${searchParams.endCursor}&sort=${searchParams.sort}&showfailed=${searchParams.showFailed}`;
+        ${RS_API_URL}/api/waters/org/${searchParams.organization}/submissions?pageSize=${searchParams.pageSize}&cursor=${searchParams.cursor}&since=${searchParams.since}&until=${searchParams.until}&sortcol=${searchParams.sortcol}&sortdir=${searchParams.sortdir}&showfailed=${searchParams.showFailed}`;
```

Comment:
Woooof, that's a big string.  I feel like we could make this a little more robust and readable by breaking it up:

```js
// using the URL API
const url = new URL('https://www.google.com/blarf');
url.searchParams.append('a', 1);
url.searchParams.append('b', 2);
...
// then reference it as such:
url.href; // evaluates to 'https://www.google.com/blah?a=1&b=2
```

Or even something as simple as using `join`s with strings:
```js
const baseUrl = 'https://www.google.com/blah';
const params = [
  'a=1',
  'b=2'
];
baseUrl + '?' + params.join('&'); //  evaluates to 'https://www.google.com/blah?a=1&b=2'
```
([The query-string package](https://github.com/sindresorhus/query-string) is helpful for simplifying this logic too!)

But I realize that this was here beforehand and that we're moving to React Query so definitely not a huge deal to address this right now!